### DISCUSSION TITLE
Upgrading to Pro

### DISCUSSION THREAD

**Discussion by henrygberry1989** [supabase/supabase#31140] [Questions] [answered]
Made an error with my database on the free account. Just wondering whether if I upgrade to Pro now whether I'll get backups from the last 7 days or if backups will start generating from today onwards?

Best, 
An absolute noob

**Comment by GaryAustin1** [comment, marked as answer]
This was my answer from a few months back... I could not find anything though on how I knew that then (I assume I've seen it here or Discord as a mod)... LOL.
https://github.com/orgs/supabase/discussions/26983
The OP did not come back and say I lied...
My thought before finding that was that they were there.
If you want a for sure answer support would be it unless you find a post I did not confirming it...

**Reply by henrygberry1989** [reply]
Thanks Gary, desperately hoping that this will be true!

**Reply by henrygberry1989** [reply]
Hey Gary wanted to say thanks for the help! Took a shot in the dark and upgraded to Pro and thankfully had 7 days that I could restore to. Means the vast majority of progress I kept.
